packages/core/src/create-lighthouse-benchmark.ts

```diff
@@ -330,6 +330,7 @@ class LighthouseSampler implements BenchmarkSampler<NavigationSample> {
 
   getMobileSettings({ width, height }: { width: number; height: number }): any {
     return {
+      extends: 'lighthouse:default',
```

Comment:
The default lighthouse configs are the mobile configs and can be extended by adding this line.